server/app/views/applicant/ApplicantLayout.java

```diff
@@ -479,24 +481,46 @@ protected DivTag renderProgramApplicationTitleAndProgressIndicator(
       int totalBlockCount,
       boolean forSummary,
       Messages messages) {
-    int percentComplete = getPercentComplete(blockIndex, totalBlockCount, forSummary);
 
-    DivTag progressInner =
-        div()
-            .withClasses(
-                BaseStyles.BG_CIVIFORM_BLUE,
+    double numerator = forSummary ? blockIndex : blockIndex + 1;
+    double denominator = forSummary ? totalBlockCount : totalBlockCount + 1;
```

Comment:
(blocking) These names no long match their purpose. Please rename to make it easier to follow.
(blocking) These don't need to be doubles. Make them int will simplify the casting later.